### PR TITLE
Soup

### PR DESCRIPTION
**Pull request by MazeChaZer**
The BeautifulSoup query works now :tada: 
The problem was that it got stuck in whitespace elements.

### CODE REVIEW

**Comment by ROYALBEFF** [comment]
Awesome. There is just a small thing I'd like to change. In the find_room function, the return statement is defined over multiple lines and with parentheses. You actually can use parantheses in your return statement, but it's not conform to the python style conventions. I'd merge the changes to the master branch, changing the return statement of the find_room function ;)

File: Pipfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "64a4b149949eab796895d1a5c3cb713100b5d13be4f4e6171db51ac7d68f3138"
+            "sha256": "68ff6a56558a01e949cd70a0f010dc4f4d64448df6eaceca52c5b6aa6ed49125"
```
Comment (dragetd) [inline Pipfile.lock:4]: I still do not understand the reason why this hash changes. Is this a hash of dep hashes? No reason to not accept the PR, but interesting nonetheless.

**Comment by dragetd** [comment]
It is a striaght forward implementation, exactly what a good hacky prototype needs to be. I like!

Why is it still draft? :)

File: Pipfile.lock (hunk 1 above)
Comment (MazeChaZer) [reply Pipfile.lock:4]: Yeah I'm also wondering about that

**Comment by MazeChaZer** [comment]
@ROYALBEFF Parentheses in `return` removed :heavy_check_mark: 
@dragetd You're right, this can be merged from my point of view

**Pull request merged**